compiler/rustc_privacy/src/lib.rs

```diff
@@ -1475,12 +1475,31 @@ impl SearchInterfaceForPrivateItemsVisitor<'_> {
                 }
             };
 
+            // FIXME: this code was adapted from the above `vis_descr` computation,
+            //        but it's not clear if it's correct.
```

Comment:
I think `fn vis_to_string` should be usable here.

Reply:
I meant `ty::Visibility::to_string`, not `vis_to_string` from rustc_ast_pretty, sorry.

`ty::Visibility::to_string` was also called `vis_to_string` when this comment was written, but it was renamed in https://github.com/rust-lang/rust/pull/115993.